Sources/PointFree/Session.swift

```diff
@@ -70,3 +72,22 @@ extension Flash: Equatable {
 }
 
 private let pointFreeUserSession = "pf_session"
+
+private func setCookie<A: Encodable>(key: String, value: A, options: Set<Response.Header.CookieOption> = []) -> Response.Header? {
+  switch AppEnvironment.current.cookieTransform {
+  case .plaintext:
+    return (try? JSONEncoder().encode(value))
```

Comment:
we should prob make a `setCookie` that takes an encodable just like we have for `setSignedCookie`